File: src/1-500/58/LengthOfLastWord.java
class Solution {
    public int lengthOfLastWord(String s) {
        if (s == null || s.length() == 0) {
            return 0;
        }
        s = s.trim();
        int pos = s.length() -1;
        while(pos >= 0) {
            if (s.charAt(pos) == ' ') {
                break;
            }
            pos--;
        }
        return s.length() - pos - 1;
    }
}

File: src/1-500/058/LengthofLastWord.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        for index in range(len(s) - 1, -1, -1):
            if s[index] == ' ':
                if count == 0:
                    continue
                else:
                    return count
            else:
                count += 1
        return count


solution = Solution()
print(solution.lengthOfLastWord("Hello world"))